src/formatDate.js

```diff
@@ -50,7 +50,48 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const dateList = date.split(fromFormat[3]);
```

Comment:
Use destructuring to get day, month, and year from dateList array.

Reply:
I have no idea how correctly write this, because the order is not the same